### ISSUE TITLE
Signature in unlock script fails to validate

### ISSUE THREAD

**Issue by kchulj** [bosagora/agora#1616]
This was observed when sending txs from faucet to a live node. The node rejects txs due to failed unlock script validation.
```
Info [agora.node.Ledger] - Rejected tx. Reason: LockType.Key signature in unlock script failed validation. Tx: { type: TxType.Payment, inputs : [{ utxo: 0x915d35...
```

**Comment by AndrejMitrovic** [comment]
Isn't this a Faucet issue?

**Comment by AndrejMitrovic** [comment]
It looks like Faucet is using ba0575a55f722c7d3a1cca90630475a652aec39d in `v0.x.x`. Have you tried updating the agora submodule?

**Comment by kchulj** [comment]
> Isn't this a Faucet issue?

Yes, so it should probably be moved..

**Comment by kchulj** [comment]
Resolved by [faucet#36](https://github.com/bpfkorea/faucet/pull/36).
